wrappers/opencv/latency-tool/rs-latency-tool.cpp

```diff
@@ -1,6 +1,7 @@
 // License: Apache 2.0. See LICENSE file in root directory.
 // Copyright(c) 2017 Intel Corporation. All Rights Reserved.
 
+#include <opencv2/highgui/highgui_c.h>
```

Comment:
C API is deprecated. Better replace everywhere
cvGetWindowHandle(window_name)
with
getWindowProperty(window_name, WND_PROP_AUTOSIZE) >= 0
to avoid future incompatibilities